packages/cli/src/util/deploy/process-deployment.ts

```diff
@@ -162,33 +166,47 @@ export default async function processDeployment({
       }
 
       if (event.type === 'created') {
+        const deployment: Deployment = event.payload;
+
         await linkFolderToProject(
           client,
           cwd,
           {
             orgId: org.id,
-            projectId: event.payload.projectId,
+            projectId: deployment.projectId!,
```

Comment:
I had a check in here that would error if `deployment.projectId` was missing, but wasn't sure if that was save given current usage or possibly old deployments. The tests didn't like it either.

I don't love using `deployment.projectId!` here, but the rest of this bit of code is more typesafe than the rampant `any` usage before.

Reply:
This seems like code smell. 

I think we've discussed this before with other PRs where `Deployment` type was incorrect based on the API response (I think @trek perhaps)

Reply:
It uses `/v10/deployments` and `projectId` appears to always be available.